### ISSUE TITLE
Add Screenshot fails because of read-only fs

### ISSUE THREAD

**Issue by diversys** [HaikuArchives/HaikuThemeManager#18]
![image](https://user-images.githubusercontent.com/1316513/55291302-d48ae780-53e5-11e9-9680-06a9f7628527.png)

**Comment by mmuman** [comment]
Yes, system themes are from packages and thus readonly. Some buttons should probably reflect this.

**Comment by Briseur** [comment]
To avoid that, packaged themes should all provide a screenshot.
Then a screenshot will only be needed if you modified them. In this case, the "new" theme should be created and appear in User Themes section as Name_of_theme (customized) and have files stored with other user themes.
Then error message could say that packaged themes can't be modified and offer the user to create a customized version
